lib/main_common.pm

```diff
@@ -2400,117 +2400,109 @@ sub load_virtualization_tests {
-        loadtest 'virtualization/universal/hotplugging_vCPUs';    # Add and remove guests vCPU
-        loadtest 'virtualization/universal/hotplugging_memory';    # Live memory change of guests
-        loadtest 'virtualization/universal/hotplugging_cleanup';    # Restore guests properties
-    }
-
-    if (check_var('VIRT_PART', 'networking')) {
-        loadtest "virt_autotest/libvirt_host_bridge_virtual_network";
-        loadtest "virt_autotest/libvirt_nated_virtual_network";
-        loadtest "virt_autotest/libvirt_isolated_virtual_network";
-    }
-
-    if (check_var('VIRT_PART', 'irqbalance')) {
-        loadtest "virt_autotest/xen_guest_irqbalance";
-    }
-
-    if (check_var('VIRT_PART', 'snapshots')) {
-        loadtest "virt_autotest/virsh_internal_snapshot";
-        loadtest "virt_autotest/virsh_external_snapshot";
-
+    if (check_var('ENABLE_HOST_INSTALLATION', 1)) {
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/prepare_profile";
+        }
+        loadtest "boot/boot_from_pxe";
+        if (get_var('AUTOYAST')) {
+            loadtest "autoyast/installation";
+        } else {
+            loadtest "installation/welcome";
+            loadtest "installation/scc_registration";
+            loadtest "installation/addon_products_sle";
+            loadtest "installation/system_role";
+            loadtest "installation/partitioning";
+            loadtest "installation/partitioning_firstdisk";
+            loadtest "installation/partitioning_finish";
+            loadtest "installation/installer_timezone";
+            loadtest "installation/user_settings";
+            loadtest "installation/user_settings_root";
+            loadtest "installation/resolve_dependency_issues";
+            loadtest "installation/installation_overview";
+            loadtest "installation/disable_grub_timeout";
+            loadtest "installation/start_install";
+            loadtest "installation/await_install";
+            loadtest "installation/reboot_after_installation";
+        }
     }
 
-    if (check_var('VIRT_PART', 'storage')) {
-        loadtest 'virtualization/universal/storage';    # Storage pool / volume test
-    }
+    if (check_var('ENABLE_VM_INSTALL', 1)) {
+        loadtest "virt_autotest/login_console";
+        loadtest "virtualization/universal/prepare_guests";
+        loadtest "virtualization/universal/waitfor_guests";
+        if (check_var('PATCH_WITH_ZYPPER', 1)) {
+            loadtest "virtualization/universal/patch_and_reboot";
+            if (my $update_package = get_var('UPDATE_PACKAGE')) {
+                if ($update_package eq 'kernel-default') {
+                    loadtest "virt_autotest/login_console";
+                    loadtest "virtualization/universal/list_guests";
+                    loadtest "virtualization/universal/patch_guests";
+                } elsif ($update_package eq 'xen' || $update_package eq 'qemu') {
```

Comment:
The UPDATE_PACKAGE list will be below. It seems to me that all other packages than kernel would need these two modules. Right?

![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/5440789/9a2c47fb-800f-40c4-b181-1a00e4a8b4f3)

Reply:
yes, from old schedule logic, that's it.
http://openqa.qam.suse.cz/tests/68283#
if need, I can make change.

Reply:
I noticed from schedule/qam/common/virt/qam_virt_install_guest.yaml that what you give now matches exactly that file. Then it is fine. Let's keep it the way it is.